.github/github_workflow_scripts/utils.py

```diff
@@ -248,3 +247,25 @@ def get_doc_reviewer(content_roles: dict[str, Any]) -> str:
     if not (reviewer := content_roles.get(DOC_REVIEWER_KEY)):
         raise ValueError("Cannot get doc reviewer")
     return reviewer
+
+
+def get_content_roles() -> dict[str, Any]:
+    """
+    Helper method to retrieve the content roles config.
+
+    Returns:
+    - `dict[str, Any]` representing the content roles. See `.github/content_roles.json` for
+    the expected structure.
+    """
+
+    print(f"Attempting to get {CONTENT_ROLES_BLOB_MASTER_URL}...")
+
+    try:
+        response = requests.get(CONTENT_ROLES_BLOB_MASTER_URL)
+        response.raise_for_status()  # Raise an error for bad status codes
+        json_data = response.json()
+        print("Successfully retrieved content_roles.json")
+        return json_data
+    except (requests.RequestException, requests.HTTPError, json.JSONDecodeError, TypeError) as e:
+        print(f"{e.__class__.__name__} getting content_roles.json: {e}")
+        return {}
```

Comment:
Why don't ewe want to raise the exception directly?

Reply:
Discussed with @RotemAmit. We will attempt to get the resource from the blob. In case we fail, we will try to get it from the filesystem instead.